libs/vkd3d/resource.c

```diff
@@ -1211,6 +1212,227 @@ HRESULT vkd3d_get_image_allocation_info(struct d3d12_device *device,
     return hr;
 }
 
+struct vkd3d_view_key
+{
+    enum vkd3d_view_type view_type;
+    union
+    {
+        struct vkd3d_buffer_view_desc buffer;
+        struct vkd3d_texture_view_desc texture;
+        D3D12_SAMPLER_DESC sampler;
+    } u;
```

Comment:
Aren't nameless unions supposed to be actually nameless now? That was my impression after PR #142.

Reply:
Doesn't matter for stuff internally, that was just for the D3D12 headers.